docs/apps/interchain-accounts/active-channels.md

```diff
@@ -9,7 +9,7 @@ The Interchain Accounts module uses [ORDERED channels](https://github.com/cosmos
 In the case of a channel closing, a controller chain needs to be able to regain access to the interchain account registered on this channel. `Active Channels` enable this functionality. Future versions of the ICS-27 protocol and the Interchain Accounts module will likely use a new 
```

Comment:
Is the note at the end here still valid, "Future versions of ICS-27 protocol and the Interchain Accounts module will likely use a new ..." The link for `Ordered Channels` suggests there's a possible flag to allow that channel to remain open `ordered_allow_timeout channels`. I apologize if I misunderstood the reference document. I'm new to IBC and ICA

Reply:
Hey @anthonyra, thanks for raising this. To answer your question:

Yes, the note here is still valid. The current implementation in `ibc-go/v6` still remains to use `ORDERED` channels. IBC core has yet to be updated to accommodate the `ORDERED_ALLOW_TIMEOUT` channel type, but when this work is implemented we will be sure to leverage it in ICA.

Please see:
- https://github.com/cosmos/ibc-go/issues/1661
- https://github.com/cosmos/ibc-go/issues/1662
- https://github.com/cosmos/ibc-go/issues/1663

This will also likely require Channel Upgrades - https://github.com/cosmos/ibc-go/issues/1599 to make this all happen!

Reply:
Awesome! I'll take a peek at all of those to get a better understanding. I wonder if the document linked should have references to those as well to indicate a work in progress? Thank you for the guidance!

Reply:
nit: I think @anthonyra 's suggestion could make sense, to add a link to the WIP tracking issues for order timeouts and channel upgrades. but this can also be added later.

Reply:
Done! Thanks for the feedback @anthonyra and @charleenfei :)